From 965907d8d7ecd777600384aee18831da1c927323 Mon Sep 17 00:00:00 2001
From: Claudia Pellegrino <claui@users.noreply.github.com>
Date: Mon, 8 Sep 2025 00:06:15 +0200
Subject: [PATCH] Remove stray `docs/conf.py` from wheel

Including `docs/conf.py` in a wheel pollutes the global namespace with
a bogus `docs.conf` module and causes conflicts with other packages that
do the same.

Exclude `docs` to prevent this.
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 0230898..c10e9e3 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -46,7 +46,7 @@ dev = [
 version = {attr = "mixpanel.__version__"}
 
 [tool.setuptools.packages.find]
-exclude = ["demo"]
+exclude = ["demo", "docs"]
 
 [tool.tox]
 envlist = ["py39", "py310", "py311", "py312"]
